src/wifi/model/dmg-wifi-mac.h

```diff
@@ -238,6 +238,7 @@ class DmgWifiMac : public RegularWifiMac
 
 protected:
   friend class MacLow;
+  friend class DmgWifiScheduler;
```

Comment:
1. is it necessary?
2. maybe it's better to move it to DmgApWifiMac

Reply:
I need "GetMbifs" method